File: k8s-dashboard-web/src/app/services/config.service.ts
import { Injectable } from '@angular/core';
import {HttpClient} from '@angular/common/http';

@Injectable({
  providedIn: 'root'
})
export class ConfigService {

  constructor(private httpClient: HttpClient) { }

  setConfig(payload) {
    return this.httpClient.post('/api/config/', payload);
  }

  getConfig() {
    return this.httpClient.get('/api/config/');
  }
}

File: k8s-dashboard-rest/src/main/java/tk/abhinav/k8sdashboardrest/service/ConfigService.java
package tk.abhinav.k8sdashboardrest.service;

import org.springframework.stereotype.Service;
import tk.abhinav.k8sdashboardrest.model.Config;

@Service
public class ConfigService {
    private Config config;

    public Config getConfig() {
        return config;
    }

    public void setConfig(Config config) {
        this.config = config;
    }
}
